i2c_comms/main.py

```diff
@@ -0,0 +1,15 @@
+# Sample program to test I2C receiving
+from i2c_receiver import receive_pot_readings
+from time import sleep
+
+while True:
+    try:
+        data = receive_pot_readings()
+
+        for i in range(0, len(data)):
+            print(f"| POT {i + 1} : {data[i]} | ", end="")
+        print("\r", end="")
+        sleep(0.05)
+
+    except KeyboardInterrupt:
+        break
```

Comment:
I would recommend printing an exit status to console before breaking here just so an exit doesn't pass silently. Never a bad idea to tell a user what the program is doing.